teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative('./person')
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Kindly note that the `parent_permission` argument uses a different notation (:) so its assignment would be different. Kindly update your super parameters to this.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

Reply:
Thank you, I will do so